solr/solr-ref-guide/modules/indexing-guide/pages/date-formatting-math.adoc

```diff
@@ -217,4 +217,4 @@ Unlike most local params, `op` is actually _not_ defined by any query parser (`f
 In the above example, it would find documents with indexed ranges that _contain_ (or equals) the range 2013 thru 2018.
 Multi-valued overlapping indexed ranges in a document are effectively coalesced.
 
-For a DateRangeField example use-case, see https://cwiki.apache.org/confluence/display/solr/DateRangeField[see Solr's community wiki].
+For a DateRangeField example use-case, see https://cwiki.apache.org/confluence/display/solr/DateRangeField[Solr's community wiki].
```

Comment:
it makes me sad we still link to this cwiki page, which errononoulys links back to ref guide..    Could you validate that linking to this cwiki page STILL makes sense?   We should bring in that content to the ref guide if so...

Reply:
Thanks for the review @epugh - I looked at the example in the cwiki, and I think it makes sense to move that to the ref guide.

I can take this up and adapt the cwiki content for the ref guide in a follow-up PR.